beacon-chain/slasher/service.go

```diff
@@ -152,8 +164,44 @@ func (s *Service) Status() error {
 	return nil
 }
 
+func (s *Service) waitForChainInitialization() {
+	stateChannel := make(chan *feed.Event, 1)
+	stateSub := s.serviceCfg.StateNotifier.StateFeed().Subscribe(stateChannel)
+	defer stateSub.Unsubscribe()
```

Comment:
Do you need to do this?
```suggestion
	defer stateSub.Unsubscribe()
	defer close(stateChannel)
```